viewing_party/party.py

```diff
@@ -0,0 +1,43 @@
+from collections import Counter
+
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        movie = {
+            'title':title,
+            'genre':genre,
+            'rating':rating
+        }
+        return movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+    
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for d in user_data['watchlist']:
+        if d['title'] == title:
+            user_data["watchlist"].remove(d)
+            user_data["watched"].append(d)
```

Comment:
one thing to consider here is that when we remove an item from a list we are currently loop through, it can cause us to repeat items in the list or skip over one.

in this context, since we only care about removing something once, this isn't a huge deal and won't necessarily mess anything up, but we could add a `break` statement after line 26 to stop the for loop once we've hit our mark.

we might also want to `append` before removing, because once removed the item is "technically" kinda gone from the list we are actively iterating through. we could do this:

```suggestion
    for d in user_data['watchlist']:
        if d['title'] == title:
            user_data["watched"].append(d)
            user_data["watchlist"].remove(d)
            break
```

or

```suggestion
    for d in user_data['watchlist']:
        if d['title'] == title:
            user_data["watched"].append(user_data["watchlist"].remove(d))
            break
```

why do you think we can pass `user_data["watchlist"].remove(d)` as an argument for `append`?